docker-compose.yml

```diff
@@ -523,8 +523,8 @@ services:
       - _APP_USAGE_STATS
       - _APP_DOCKER_HUB_USERNAME
       - _APP_DOCKER_HUB_PASSWORD
-      - _APP_LOGGING_CONFIG
```

Comment:
We should remove all occurrences of _APP_LOGGING_PROVIDER from the compose file as well as compose.phtml